app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -102,8 +99,9 @@ class HomeScreenRobot {
         mDevice.wait(Until.findObject(By.res("org.mozilla.fenix.debug:id/collections_header")), waitingTime)
     }
     fun typeCollectionName(name: String) {
-        mDevice.wait(Until.findObject(By.res("org.mozilla.fenix.debug:id/name_collection_edittext")), waitingTime)
-        collectionNameTextField().check(matches(hasFocus()))
+        mDevice.wait(Until.findObject(By.res("name_collection_edittext")), waitingTime)
```

Comment:
same as above

Reply:
for some reason that I don't know the tests do not work with `mDevice.waitNotNull` that's why I left this one this way.